config/currency_iso.json

```diff
@@ -1253,7 +1253,7 @@
     "subunit": "Cent",
     "subunit_to_unit": 100,
     "symbol_first": false,
-    "html_entity": "&#x0BF9;",
+    "html_entity": "₨",
```

Comment:
You are right, the previous entity was wrong, but the new one should be `&#8360;`, not a unicode character